File: app/src/main/java/com/pr0gramm/app/util/BackgroundScheduler.java
package com.pr0gramm.app.util;

import rx.Scheduler;
import rx.schedulers.Schedulers;

/**
 * A scheduler that put work on some background scheduler
 */
public class BackgroundScheduler {
    public static Scheduler instance() {
        return Schedulers.io();
    }
}

File: app/src/main/java/com/pr0gramm/app/util/BackgroundScheduler.kt
package com.pr0gramm.app.util

import rx.Scheduler
import rx.android.schedulers.AndroidSchedulers
import rx.schedulers.Schedulers

/**
 * A scheduler that put work on some background scheduler
 */
val BackgroundScheduler: Scheduler = Schedulers.io()

/**
 * A scheduler to run work on the main thread.
 */
val MainThreadScheduler: Scheduler = AndroidSchedulers.mainThread()